README.md

```diff
@@ -59,13 +59,15 @@ grid = ImmersedBoundaryGrid(grid, GridFittedBottom(bathymetry))
 
 # Build an ocean simulation initialized to the ECCO state estimate on Jan 1, 1993
 ocean = ClimaOcean.ocean_simulation(grid)
-dates = DateTimeProlepticGregorian(1993, 1, 1)
-set!(ocean.model, T = ClimaOcean.ECCOMetadata(:temperature; dates),
-                  S = ClimaOcean.ECCOMetadata(:salinity; dates))
+dates = DateTime(1993, 1, 1) : Month(1) : DateTime(1994, 1, 1)
+set!(ocean.model,
+     T=ClimaOcean.Metadata(:temperature; dates=first(dates), dataset=ClimaOcean.ECCO4Monthly()),
+     S=ClimaOcean.Metadata(:salinity;    dates=first(dates), dataset=ClimaOcean.ECCO4Monthly()))
 
 # Build and run an OceanSeaIceModel (with no sea ice component) forced by JRA55 reanalysis
 atmosphere = ClimaOcean.JRA55PrescribedAtmosphere(arch)
-coupled_model = ClimaOcean.OceanSeaIceModel(ocean; atmosphere)
+radiation  = ClimaOcean.Radiation(arch)
```

Comment:
this is necessary? Can we make it not?

Reply:
Code was breaking without it.
We can make it the default in `ClimaOcean.OceanSeaIceModel`?

Reply:
Other ideas welcome

Reply:
For sure cause it doesn’t add anything useful to build it here does it

Reply:
done